addons/mail/static/src/views/fields/statusbar_time/statusbar_time_field.js

```diff
@@ -0,0 +1,114 @@
+/** @odoo-module **/
+
+import { _lt } from "@web/core/l10n/translation";
+import {
+    preloadStatusBar,
+    statusBarField,
+    StatusBarField,
+} from "@web/views/fields/statusbar/statusbar_field";
+import { registry } from "@web/core/registry";
+
+export class StatusBarTimeField extends StatusBarField {
```

Comment:
Would go for a "Duration" naming instead of "Time", as we are more tracking duration than precise hours. In odoo we generally use duration for that kind of computation.